tools/ios/package_ops.config

```diff
@@ -1 +1 @@
-com.microsoft.extensions;1;DecodeImage,EncodeImage,BertTokenizer
+com.microsoft.extensions;1;DecodeImage,EncodeImage,BertTokenizer,ClipTokenizer,BpeDecoder,DrawBoundingBoxes
```

Comment:
Not interesting in AudioDecoder op?

Reply:
How big is it?

Reply:
x64 dll delta is around 250K, compressed size less than 100k.
I think Android and iOS CI pipelines by default include all custom ops, only release pipelines are using selected op config file.

Reply:
All custom op input/output are described in this Python class. I knew it would be better there are some docs for that, probably we can auto-gen it from the source.

https://github.com/microsoft/onnxruntime-extensions/blob/2fa0b710eac5d39088402e369965cad126575d71/onnxruntime_extensions/_cuops.py#LL393C10-L393C10

Most of custom op unit tests are written in Python, Android/iOS only builds the source code to make sure it be compatible with the compiler and dependencies. Few of them have some C++ tests, not sure how convenient it is to run on Android/iOS.

Reply:
Have all the open source licenses for things used by the AudioDecoder op both directly and indirectly been reviewed and added to the open source registrations for extensions?

Reply:
Will discuss offline and create separate review for any additional ops. May also want to include more tokenizers.